File: apps/s-2-admin/src/api/availability/Availability.ts
import { User } from "../user/User";
import { EventType } from "../eventType/EventType";
import { Schedule } from "../schedule/Schedule";

export type Availability = {
  id: number;
  user?: User | null;
  eventType?: EventType | null;
  days: number;
  startTime: Date;
  endTime: Date;
  date: Date | null;
  schedule?: Schedule | null;
};

File: apps/cactus-demo-dotnet/src/APIs/Availability/Dtos/Availability.cs
namespace CactusDemoDotnet.APIs.Dtos;

public class Availability
{
    public int Id { get; set; }

    public int? User { get; set; }

    public int? EventType { get; set; }

    public int Days { get; set; }

    public DateTime StartTime { get; set; }

    public DateTime EndTime { get; set; }

    public DateTime? Date { get; set; }

    public int? Schedule { get; set; }
}
